### PR TITLE
Updated deps, fixes type error

### PR DESCRIPTION
**Pull request by jrson83**
Hi,
I wanted to use the plugin on the lume.land site and it was throwing a type error.
I updated the deps to fix the error. Can you please release the new version?

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,19 @@ and this project tries to adhere to
 	**Security** in case of vulnerabilities.
 -->
 
+## 0.2.1 - 2022-10-11
+
+### Fixed
+
+- Argument of type '(site: Site) => void' is not assignable to parameter of type
+  'Plugin'.
```
Comment (binyamin) [inline CHANGELOG.md]:
```suggestion
- Fixed type-error by updating Lume
```

**Comment by binyamin** [comment]
Thanks so much. I reworded the changelog entry, slightly. As long as you're okay with the changes, I'll go ahead and merge this.

File: CHANGELOG.md (hunk 1 above)
Comment (jrson83) [reply CHANGELOG.md]: Sure , I will update the PR later today.

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,6 +15,18 @@ and this project tries to adhere to
 	**Security** in case of vulnerabilities.
 -->
 
+## 0.2.1 - 2022-10-11
```
Comment (binyamin) [inline CHANGELOG.md:18]:
```suggestion
## 0.2.1 - Unreleased
```

**Pull request merged**